package-lock.json

```diff
@@ -13226,10 +13281,24 @@
         "walker": "~1.0.5"
       }
     },
+    "sanitizer": {
```

Comment:
We didn't last long without sanitizer did we :D

Reply:
Opps